### PR TITLE
Feature/cfa 314 add training layouts and css

### PR DESCRIPTION
**Pull request by NourJammal**
CFA
Showcase 
Add training details layouts and css

https://smartjira.atlassian.net/browse/CFA-314

### CODE REVIEW

File: src/Showcase/src/Smart.FA.Catalog.Showcase.Infrastructure/Data/Scripts/v_TrainingDetails.sql [hunk 1]
```diff
@@ -1,6 +1,7 @@
 CREATE VIEW [dbo].[v_TrainingDetails]
 AS
-SELECT training.Id, detail.Title, detail.Methodology, detail.Goal, category.TrainingTopicId, detail.Language, trainer.FirstName, trainer.LastName, training.StatusId
+SELECT training.Id, detail.Title, detail.Methodology, detail.Goal, detail.PracticalModalities, category.TrainingTopicId, detail.Language, trainer.FirstName,
+trainer.LastName,trainer.Title AS trainerTitle, training.StatusId
```
Comment (maxime-poulain) [inline src/Showcase/src/Smart.FA.Catalog.Showcase.Infrastructure/Data/Scripts/v_TrainingDetails.sql:4]: For better readability you can apply the following formatting to the select part
image

File: src/Showcase/src/Smart.FA.Catalog.Showcase.Web/Pages/Training/TrainingDetails/TrainingDetails.cshtml [hunk 2]
```diff
@@ -6,30 +6,58 @@
...
+            </div>
+            <div class="training-card__details-section">
+                <div class="training-card__details-label">
+                    Objectifs
+                </div>
+                @Model.Training.TrainingGoal
+            </div>
+            <div class="training-card__details-section">
+                <div class="training-card__details-label">
+                    Modalités pratiques
```
Comment (maxime-poulain) [inline src/Showcase/src/Smart.FA.Catalog.Showcase.Web/Pages/Training/TrainingDetails/TrainingDetails.cshtml:40]: You can add this to the resources(translations).

**Review by maxime-poulain — CHANGES_REQUESTED** [review]

File: src/Showcase/src/Smart.FA.Catalog.Showcase.Web/Pages/Training/TrainingDetails/TrainingDetails.cshtml (hunk 2 above)
Comment (NourJammal) [reply src/Showcase/src/Smart.FA.Catalog.Showcase.Web/Pages/Training/TrainingDetails/TrainingDetails.cshtml:40]: Pour la v1 on a que le français 
On créera un ticket pour la creation des fichiers resources

Comment (maxime-poulain) [reply src/Showcase/src/Smart.FA.Catalog.Showcase.Web/Pages/Training/TrainingDetails/TrainingDetails.cshtml:40]: Oki

**Review by maxime-poulain — APPROVED** [review]

File: src/Showcase/src/Smart.FA.Catalog.Showcase.Web/Pages/Training/TrainingDetails/TrainingDetails.cshtml [hunk 3]
```diff
@@ -6,30 +6,58 @@
...
+                </div>
+                @Model.Training.TrainingStatus
```
Comment (maxime-poulain) [inline src/Showcase/src/Smart.FA.Catalog.Showcase.Web/Pages/Training/TrainingDetails/TrainingDetails.cshtml:42]:
```suggestion
                @Model.Training.TrainingPracticalModalities
```

**Review by maxime-poulain — CHANGES_REQUESTED** [review]

File: src/Showcase/src/Smart.FA.Catalog.Showcase.Web/Pages/Training/TrainingDetails/TrainingDetails.cshtml (hunk 2 above)
Comment (joe-coop) [reply src/Showcase/src/Smart.FA.Catalog.Showcase.Web/Pages/Training/TrainingDetails/TrainingDetails.cshtml]: I will add localization keys for translation https://smartjira.atlassian.net/browse/CFA-349

**Pull request merged**